README.md

```diff
@@ -1,33 +1,143 @@
+- Extensions provide abstraction from the orchestrator, some resemble known APIs to facilitate integration
+- Ability to leverage Linux, Windows, CPU, GPU resources
+- Cloud provider agnostic while utilizing cloud provider services  
+- Activity log visualization via fluent-bit connectors
+- Resource and task monitoring via Grafana, Prometheus, Admin GUI
+
+Use cases include:
+
+- Scientific computing / HPC
+- Financial analysis
+- Machine learning / data processing   
+- Video encoding / transcoding 
+- Genomics / bioinformatics
+- Rendering / graphics
+- Any batch processing workload
+
+## Want a quickstart?
+
+Get started with ArmoniK by following the quickstart guide:
+
+[Quickstart guide](https://armonik.readthedocs.io/en/latest/installation/linux/deployment/)
+
+Or try deploying ArmoniK locally on your machine.
+
+## Features
+
+- Horizontally scalable architecture  
+- gRPC APIs for task management (11 language clients)
+- Admin GUI for monitoring
+- Enterprise security compliance
+- Mutual TLS authentication
+- Partitioning of resources
+- Extensions for simplified integration
+- Multi-cloud and on-premise support 
+- Detailed monitoring with Grafana, Prometheus, etc.
+- Dynamic scaling through Kubernetes
+- Client SDKs for .NET, Python, C++, and more
+
+## Repository 
+
+The source code for ArmoniK is available on GitHub:
 
-- 👉 [ArmoniK Architecture](https://aneoconsulting.github.io/ArmoniK/armonik)
-- 👉 [ArmoniK Versions](https://aneoconsulting.github.io/ArmoniK/armonik#versions)
-- 👉 [ArmoniK Installation](https://aneoconsulting.github.io/ArmoniK/installation)
-- 👉 [ArmoniK Configuration](https://aneoconsulting.github.io/ArmoniK/guide/how-to/how-to-configure-authentication)
-- 👉 [ArmoniK Performance](https://aneoconsulting.github.io/ArmoniK/benchmarking/test-plan)
+[https://github.com/aneoconsulting/ArmoniK](https://github.com/aneoconsulting/ArmoniK)
 
+The components are spread across multiple repositories:
 
-## Bug
+- [ArmoniK.Core](https://github.com/aneoconsulting/ArmoniK.Core) - The core logic of ArmoniK
+- [ArmoniK.Api](https://github.com/aneoconsulting/ArmoniK.Api) - The gRPC APIs 
+- [ArmoniK.Extensions.Csharp](https://github.com/aneoconsulting/ArmoniK.Extensions.Csharp) - The C# extensions
+- [ArmoniK.Extensions.Cpp](https://github.com/aneoconsulting/ArmoniK.Extensions.Cpp) - The C++ extensions
 
-You are welcome to raise issues on GitHub. Please, read our [community guidelines](https://aneoconsulting.github.io/ArmoniK.Community/) before doing so.
 
-You can also send us a direct email at [armonik-support@aneo.fr](mailto:armonik-support@aneo.fr).
+## Releases
 
-## Acknowledge
+The latest ArmoniK releases are available on [GitHub](https://github.com/aneoconsulting/ArmoniK/releases) and [Docker Hub](https://hub.docker.com/u/dockerhubaneo).
+
+
+
+## Benchmarking  
+
+Performance benchmarks for ArmoniK are available here:
+
+[Benchmark results](https://aneoconsulting.github.io/ArmoniK/benchmarking/test-plan/)
```

Comment:
This is totally obsolete and not maintain. I will not mention it is until it up to date.